ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -44,39 +45,79 @@ describe('AdminInstitutionEditSpec', () => {
+  });
+
+  it('should default DOMAINS to ADDRESSES in Controlled Tier requirement', async() => {
+    urlParamsStore.next({
+      institutionId: 'Verily'
+    });
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
+    ctEnabledToggle.props.onChange({
+      originalEvent: undefined,
+      value: true,
+      target: {name: 'name', id: '', value: true}
+    });
+    await waitOneTickAndUpdate(wrapper);
+
+    const ctAgreementTypeDropDown = wrapper.find('[data-test-id="ct-agreement-dropdown"]');
+    expect(ctAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should hide/show controlled card when controlled tier disabled/enabled', async() => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+    const testInput = fp.repeat(81, 'a');
+
+    expect(wrapper.find('[data-test-id="ct-card-container"]').exists()).toBeFalsy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
+    ctEnabledToggle.props.onChange({
+      originalEvent: undefined,
+      value: true,
+      target: {name: 'name', id: '', value: true}
+    });
+    await waitOneTickAndUpdate(wrapper);
+    expect(ctEnabledToggle.props.checked).toBeTruthy();
+    expect(wrapper.find('[data-test-id="ct-card-container"]').exists()).toBeTruthy();
   });
 
-  it('should show appropriate section after changing agreement type', async() => {
+  it('should show appropriate section after changing agreement type in Registered Tier requirement', async() => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
     const testInput = fp.repeat(81, 'a');
 
-    const agreementTypeDropDown = wrapper.find('[data-test-id="agreement-dropdown"]').instance() as Dropdown;
+    const agreementTypeDropDown = wrapper.find('[data-test-id="rt-agreement-dropdown"]').instance() as Dropdown;
     agreementTypeDropDown.props.onChange({
       originalEvent: undefined,
       value: InstitutionMembershipRequirement.DOMAINS,
       target: {name: 'name', id: '', value: InstitutionMembershipRequirement.DOMAINS}
     });
     await waitOneTickAndUpdate(wrapper);
 
-    let emailAddressDiv = wrapper.find('[data-test-id="emailAddress"]');
-    let emailDomainDiv = wrapper.find('[data-test-id="emailDomain"]');
-    expect(emailAddressDiv.length).toBe(0);
-    expect(emailDomainDiv.length).toBe(2);
+    let rtEmailAddressDiv = wrapper.find('[data-test-id="rtEmailAddress"]');
+    let rtEmailDomainDiv = wrapper.find('[data-test-id="rtEmailDomain"]');
+    expect(rtEmailAddressDiv.length).toBe(0);
+    expect(rtEmailDomainDiv.length).toBe(2);
```

Comment:
why 2?

Reply:
that one is copied from previous RT test. Let me take closer look

Reply:
Not be able to know why. Seems wrapper is able to find two div with that id even there is one is the code. Any ideas?

Reply:
OK - could be related to https://precisionmedicineinitiative.atlassian.net/browse/RW-6331